File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Dias/104.pl
print "Digite nome:\n";
$nome = <STDIN>;
print "Digite sexo:\n";
$sexo = <STDIN>;
chomp ($sexo);
print "Digite idade:\n";
$idade = <STDIN>;
if (($sexo eq "F" || $sexo eq "f") && ($idade < 25) )
{
	print "$nome ACEITA\n";
}
else
{
	print "$nome NAO ACEITA\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Dias/104.rb
puts "Digite nome:"
nome = gets
puts "Digite sexo:"
sexo = gets.chomp
puts "Digite idade:";
idade = gets
if sexo == 'F' || sexo == 'f' && idade.to_i < 25
  puts "#{nome} ACEITA"
else
  puts "#{nome} NAO ACEITA"
end